engine/citizens_advice_components.gemspec

```diff
@@ -25,5 +25,6 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency     "haml-rails", [">= 2.0.0", "< 3.0"]
   spec.add_runtime_dependency     "rails", [">= 6.0.0", "< 7.0"]
+  spec.add_runtime_dependency     "rails-i18n", [">= 6.0.0", "< 7.0"]
```

Comment:
This is a requirement of translating the date within the component using `I18n.l`